content/en/blog/2022/opamp/index.md

```diff
@@ -0,0 +1,239 @@
+---
+title: Using OpenTelemetry OpAMP to modify service telemetry on the go
+linkTitle: OpAMP service usage
+date: 2022-10-12
+author: >-
+  [Nimrod Shlagman](https://github.com/nemoshlag) (Cisco)
+---
+
+How verbose should your service telemetry be? Should a service output all
+traces, metrics, and logs 100% of the time? How much of the service traffic
+should be sampled? I would like to suggest the answer of “it depends”. Desired
+telemetry data differs in a service lifecycle from development to continuous
+deployment. It changes when a client faces an error or our service has been
```

Comment:
For consistency, please replace "our service" with "a service" (our you could instead replace "a client" with "our client", but in this case an impersonal subject is more appropriate to the sentence as it's discussing the situation in general).